libvast/include/vast/system/actors.hpp

```diff
@@ -275,6 +279,10 @@ using index_actor = typed_actor_fwd<
   // preserving the old one(s).
   caf::replies_to<atom::apply, transform_ptr, std::vector<uuid>,
                   keep_original_partition>::with<std::vector<partition_info>>,
+  // A shorthand to the above handler that runs an identity transformation on
+  // the provided partitions without keeping the originals.
```

Comment:
Imho the main reason for having a separate handler is that it allows a *client command* to trigger the transform, just having a shorthand would probably not justify a separate handler.